src/nvim/api/dispatch_deprecated.lua

```diff
@@ -7,6 +7,7 @@ local deprecated_aliases = {
   nvim_buf_get_number="buffer_get_number",
   nvim_buf_get_option="buffer_get_option",
   nvim_buf_get_var="buffer_get_var",
+  nvim_buf_live_updates="buffer_live_updates",
```

Comment:
I'd prefer to leave this backwards-compatibility in since I'd like to try and have live updates merged soon, and I want it to work with current python clients.